11-slim/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM azul/zulu-openjdk-debian:11 AS jre
+
+RUN apt-get -y update && apt-get -y install binutils
+
+# Included modules cherrypicked from https://docs.oracle.com/en/java/javase/11/docs/api/ using random
+# guessing of what should be required by a minimal zipkin-server.
+RUN cd / && jlink --no-header-files --no-man-pages --compress=2 --strip-debug \
+    --add-modules java.base,java.desktop,java.instrument,java.logging,java.management,java.management.rmi,java.sql,jdk.unsupported \
```

Comment:
we should be able to tell which modules we need.. there are a very limited number of supported integrations. It will boil down to Armeria + AWS SDK

particularly we actively disable JMX. if java.desktop is where java.beans is, we should probably make note of that in the file.

Reply:
Yeah it wasn't a good comment - we should be good with base, desktop, logging, unsupported after there is a slim jar. I think it's good to also allow instrumentation as best-effort though we could cut that too.

Reply:
fwiw I don't see a short term out of sql due to mysql

Reply:
oh right.. but which slim! yes zipkin's slim dist wouldnt need sql